install/kubernetes/helm/istio/values.yaml

```diff
@@ -28,6 +28,16 @@ mixer:
     enabled: true
   telemetry:
     enabled: true
+    averageLatencyThreshold: 75ms
```

Comment:
we probably want a comment here about why `75ms` and how this should be adjusted, etc.